### ISSUE TITLE
--pre_filter error

### ISSUE THREAD

**Issue by clemgoub** [elzbth/jitterbug#8]
Hello,

I am currently running Jitterbug on several bam files in human samples. It seems to work properly so far, and I really look forward to see the results! However I have an issue passing the parameter --pre_filter (my coverage is 30-40X and I woud like to try it), the program crashes with this error if I try to specify it:

Traceback (most recent call last):
File "jitterbug.py", line 129, in <module>
main(sys.argv[1:])
File "jitterbug.py", line 119, in main
args.numCPUs, args.bin_size, args.minMAPQ, generate_test_bam, args.pre_filter, args.conf_lib_stats, mem, args.min_cluster_size,args.step_one_only,args.step_two_only)
File "/home/cgoubert/bin/jitterbug/Run_TE_ID_reseq_streaming.py", line 38, in run_jitterbug_streaming
print proc.get_memory_info().rss
AttributeError: 'Process' object has no attribute 'get_memory_info'

Thanks a lot for your help!

Clément

**Comment by mbosio85** [comment]
Hi Clément,

I think it should be some module incompatibilities python  - linux.

Here http://stackoverflow.com/questions/20027440/psutil-module-not-fully-working-on-debian-7 they suggest to update psutil via pip

Or here . https://github.com/fabianp/memory_profiler/issues/114 . they suggest to reinstall memory profiler with pip.

Please let me know how it goes and if it gets solved. One quick fix would be to comment all line with print proc.get_memory_info().rss from Run_TE_ID_reseq_streaming.py if you are not interested in keeping track of the memory usage.

Mattia

**Comment by clemgoub** [comment]
Hi Mattia,

Thanks for your reply! I tried your two suggestions: updating psutil and reinstalling memory_profiler using pip, but that unfortunately didn't fixed the problem.

I did comment the print proc.get_memory_info().rss lines however, and so far the script seems to work.

It's no big deal for me to keep track of the memory usage, however, let me know if you have any idea.

Thanks again,

Clément
